packages/spaces/commands/vpn/info.js

```diff
@@ -52,12 +58,12 @@ module.exports = {
   description: 'display the information for VPN',
   help: `Example:
 
-    $ heroku spaces:vpn:info my-space
-    === my-space VPN Info
+    $ heroku spaces:vpn:info my-space --name vpn-connection-name
```

Comment:
I think this is wrong now that we don't allow the space as an arg (sorry for not catching this in my last round of comments!).